takecare19/takecare19
# Admin login

I already added you as a user via firebase, you can manage it here: https://console.firebase.google.com/u/0/project/takecare19-prod/authentication/users

**Testing instructions**
* add `/admin` to the url
* enter your email and your password (it is `pothos`, you can change it in the link above)
* confirm you are redirected to the "Dashboard" page
* confirm that if you refresh while on the dashboard page, you stay logged in
* confirm that you cannot access the dashboard page without logging in
